Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -220,12 +233,15 @@ class CustomerIOImplementation: CustomerIOInstance {
      is no active customer, this will fail to register the device
      */
     public func registerDeviceToken(_ deviceToken: String) {
+        CIODataPipeline.analytics.setDeviceToken(deviceToken)
```

Comment:
The function `addDeviceAttributes` is currently the common function used for when a device is registered with a profile and when adding attributes to a device.

Shouldn't this line go into that function instead?

Reply:
`addDeviceAttributes` also takes in attributes that can be added, `setDeviceToken` doesn;t currently take in any attributes, we need to create that method ourself. I'll add a todo that this method will then move there.